<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imgViewLabel">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>7</y>
      <width>691</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="langBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ballDetect">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>510</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Erkennung starten</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sprache ausw√§hlen:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="inputBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Input:</string>
    </property>
    <widget class="QRadioButton" name="webcamRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Webcam</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="bildRadio">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Bild</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="videoRadio">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Video</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="onlyoneCheck">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>490</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nur 1 Objekt anzeigen</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
